maintainers/maintainer-list.nix

```diff
@@ -692,6 +692,11 @@
     githubId = 135230;
     name = "Aycan iRiCAN";
   };
+  b42 = {
+    email = "martin@martinmilata.cz";
+    github = "mmilata";
```

Comment:
Since https://github.com/NixOS/nixpkgs/pull/66361 you need to provide `githubId` as well, see the comment at the top of this file.